frontend/src/scenes/dashboard/DashboardItem.tsx

```diff
@@ -326,12 +326,10 @@ export function DashboardItem({
                                     trigger={['click']}
                                     overlay={
                                         <Menu data-attr={'dashboard-item-' + index + '-dropdown-menu'}>
-                                            <Menu.Item
-                                                data-attr={'dashboard-item-' + index + '-dropdown-view'}
-                                                icon={<Icon />}
-                                                onClick={() => router.actions.push(link)}
-                                            >
-                                                {viewText}
+                                            <Menu.Item data-attr={'dashboard-item-' + index + '-dropdown-view'}>
+                                                <a href={link}>
```

Comment:
There exists a `<Link>` element, we should use it I think. Otherwise this is gonna cause a full-page reload.

Reply:
Great catch, slipped my mind!